tartufo/scanner.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-lines
```

Comment:
Fascinating that we're finally hitting this. Makes me think we should make a change (not necessarily in here) that has been bugging me: Moving the `Issue` class out to `types.py`, where it belongs a bit better.

Reply:
I strongly agree with moving Issue to types.py